packages/api/src/command/medical/document/document-query.ts

```diff
@@ -83,13 +86,25 @@ export async function queryDocumentsAcrossHIEs({
   const cwData = externalData as PatientDataCommonwell;
   if (!cwData.patientId) return createQueryResponse("failed");
 
-  const updatedPatient = await updateDocQuery({
+  const updatedPatientProgress: Patient = await updateDocQuery({
     patient: { id: patient.id, cxId: patient.cxId },
     downloadProgress: { status: "processing" },
     requestId,
     reset: true,
   });
 
+  const updatedPatientMetadata = await updatePatient({
```

Comment:
we're sending two updates to patient, including one that overwrites all its `data`, and ended up making `updatePatient` to have query specific logic.

I'm sending a commit with a suggestion of code to simplify it both for doc query and consolidated query.